crates/ethereum-rpc/src/lib.rs

```diff
@@ -613,7 +621,7 @@ fn register_rpc_methods<C: sov_modules_api::Context, Da: DaService>(
                     .block_number()
                     .await;
 
```

Comment:
we have two requests in this function now.

1. to the sequencer
2. to the da layer.

we can do both async, and await on them, lowering time needed to execute this function

Reply:
@kpp @rakanalh in tokio, what's the equivalent of JavaScript's Promise.all ?

Reply:
https://docs.rs/futures/0.3.5/futures/future/fn.join_all.html

Reply:
or:
https://docs.rs/tokio/latest/tokio/macro.join.html

Reply:
done but I cloned ethereum.sequencer_client, if it is bad practice I can find a workaround